### ISSUE TITLE
IMPORT_SEED_RELAYS required even if IMPORT_SEED_RELAYS_FILE is provided

### ISSUE THREAD

**Issue by eleaner** [bitvora/haven#27]
how does it work
does content of the file overlay the IMPORT_SEED_RELAYS?

it is a bit confusing and I guess the same applies to BLASTR_RELAYS

**Comment by eleaner** [comment]
actually i am not sure the file works at all
e.g it does not seem to blastr to relays I added

**Comment by eleaner** [comment]
correction,
i think it works, but it stops blasting on first error reported by any relay

**Comment by HolgerHatGarKeineNode** [comment]
I fixed this with #25 

Only the string from the .env is currently used in the code. The file is only accepted if the string is left empty.

Because of this if statement:

```go
if envValue != "" {
  return getRelayList(envValue)
}
```

The PR #25 changes this behavior to:

```go
if filePath != "" {
  if _, err := os.Stat(filePath); err == nil {
    return getRelayListFromFile(filePath)
  }
}

if envValue != "" {
  return getRelayList(envValue)
}
```

**Comment by eleaner** [comment]
awesome!
few more questions
- it seems like the list of relays is read from file and held in memory = to update the file content one has to restart haven. Would it might be a good idea (low priority) to reload the content on change?
- should I see in the log information about successful blast to all the listed relays? I still suspect something is not working in my setup and it is not blasting as expected - maybe (but I am not sure how to check it) just fails on first error and stops blasting to subsequent relays.

**Comment by HolgerHatGarKeineNode** [comment]
Yes, you're right, I think the config is loaded from the .env every time main.go is restarted. So this is also a possible JSON file.

Everything else would be a new feature request and would have to be implemented with GO.

For now, it only works with a restart after changing the relay lists.

I have also experimented and simply put the entire list from nostr.watch into the JSON. I also noticed that blastr hangs if there are too many relay errors.

Especially all the relays that only require AUTH throw out a lot of errors.

**Comment by eleaner** [comment]
@HolgerHatGarKeineNode 
any news on handling the blastr errors?

**Comment by HolgerHatGarKeineNode** [comment]
I'm not working on that. Just added a few Docker things as PR.

**Comment by barrydeen** [comment]
Should be fixed now in #32 , please pull the latest
